ydb/public/tools/lib/cmds/ut/test.py

```diff
@@ -24,3 +24,45 @@ def test_kikimr_config_generator_generic_connector_config():
 
     actual = generic_connector_config()
     assert actual == expected
+
+def test_merge_two_yaml_configs():
+    patched_yaml = yatest.common.output_path("patched.yaml")
+
+    with open(yatest.common.source_path("ydb/public/tools/lib/cmds/ut/config.yaml"), 'r') as fh:
+        data_1 = yaml.load(fh, Loader=yaml.FullLoader)
+    
+    with open(yatest.common.source_path("ydb/public/tools/lib/cmds/ut/patch.yaml"), 'r') as fh:
+        data_2 = yaml.load(fh, Loader=yaml.FullLoader)
+
+    with open(patched_yaml, "w") as res:
+        res.write(yaml.dump(merge_two_yaml_configs(data_1, data_2), default_flow_style=False))
+
+    return yatest.common.canonical_file(patched_yaml, local=True)
+
+def test_merge_two_dicts():
+    dict_1 = {'data': [{'user': 'root', 'password': '1234'}]}
```

Comment:
see to https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions for test parameters